lib/src/markdown_processor.dart

```diff
@@ -369,26 +357,25 @@ MatchingLinkResult _getMatchingLinkElementLegacy(String codeRef,
 }
 
 /// Given a set of commentRefs, return the one whose name matches the codeRef.
```

Comment:
Looks like the doc comment doesn't apply any more.

Reply:
Wrote a more complete doc.